draft-ietf-rats-architecture.md

```diff
@@ -973,28 +973,40 @@ A more detailed discussion with examples appears in {{time-considerations}}.
 
 # Privacy Considerations
 
-The conveyance of Evidence and the resulting Attestation Results
-reveal a great deal of information about the internal state of a
-device.  In many cases, the whole point of the Attestation process is
-to provide reliable information about the type of the device and the
-firmware/software that the device is running.  This information might be
-particularly interesting to many attackers. For example, knowing that a device is
-running a weak version of firmware provides a way to aim attacks better.
-
-Evidence and Attestation Results data structures are expected to support
-integrity protection encoding (e.g., COSE, JOSE, X.509) and optionally might
-support confidentiality protection (e.g., COSE, JOSE).
-Therefore, if confidentiality protection is omitted or unavailable, the protocols
-that convey Evidence or Attestation Results are responsible for detailing what
-kinds of information are disclosed, and to whom they are exposed.
-
-Furthermore, because Evidence might contain sensitive information,
-Attesters are responsible for only sending such Evidence to trusted
-Verifiers.  Some Attesters might want a stronger level of assurance of
-the trustworthiness of a Verifier before sending Evidence to it.  In such cases,
-an Attester can first act as a Relying Party and ask for the Verifier's own
-Attestation Result, and appraising it just as a Relying Party would appraise
-an Attestation Result for any other purpose.
+Every claim in Attestation Evidence and Attestation Results is potentially PII (Personally Identifying Information) depending on the end-end use case of the attestation.
```

Comment:
```suggestion
Every Claim in attestation Evidence and Attestation Results is potentially PII (Personally Identifying Information) depending on the end-end use case of the remote attestation procedure.
```

Reply:
I don't think it's true that every claim is PII.  If someone adds a claim saying it supports a specific version of the EAT spec, just like every other device, I wouldn't call that PII.  Similarly if there is a claim that contains the current time, or a nonce supplied by a verifier, those wouldn't be PII either under any use case I can think of.   The key question is whether it significantly narrows the anonymity set (as discussed in RFC 6973).

Reply:
I also think the original text in red needs to stay at the top.  PII is not the only aspect of privacy, it's just an important subset.

Reply:
Strike "Every", change "Claim" to "Claims" and change "is" to "are". Then I'm OK with the proposed new text.